### PR TITLE
fix: Fixed overflow issue inside the ldap page

### LINKED ISSUE #21178: Login toggle gets pushed off the screen
### Describe the Improvement

When enabling LDAP a new toggle appears on the login page. This toggle however gets pushed partly off the screen when using a long translated title.

In Firefox:
image

Chrome seems to handle it better although it does it does still look somewhat funky:
image

Looking at the screenshots perhaps the Login button should be limited in its max width too.

### PR DESCRIPTION
**Pull request by Julias0**
## Scope

What's changed:

- Have added support for text inside the sign in button to be truncated when the internationalization causes the content
of sign in to overflow hiding the provider selector in the login page

## Potential Risks / Drawbacks

- The whole content of sign in cannot be seen unless its hovered over if its too long. But I guess this shouldnt be  a problem since folks mostly know what the primary CTA does in a login page

## Review Notes / Questions
Heres a screenshot of how it looks post fix. I had changed a few variables in the login page to get this state, so ignore if something in providers looks different than how it should
Screenshot 2024-09-26 at 9 10 01 PM

---

Fixes #21178

### CODE REVIEW

**Comment by hanneskuettner** [comment]
Thank you for the PR! 

Did you also take a look at the other overflow issues displayed in the initial PR?

**Comment by Julias0** [comment]
> Thank you for the PR!
> 
> Did you also take a look at the other overflow issues displayed in the initial PR?

Hey yes. Should I fix them off in the same pattern? I was worried about truncating too many things, sign in buttons are easier to make out even after truncation

**Comment by hanneskuettner** [comment]
Let's bring them all into line. The truncation should really only be a last resort if the translations really are too long and seldomly visible in normal operation.

File: app/src/routes/login/components/login-form/ldap-form.vue [hunk 1]
```diff
@@ -118,4 +120,14 @@ async function onSubmit() {
 .v-notice {
 	margin-bottom: 20px;
 }
+
+.sign-in-ellipsis {
```
Comment (hanneskuettner) [inline app/src/routes/login/components/login-form/ldap-form.vue:124]: Let's just use `.sign-in` as class name.

**Comment by Julias0** [comment]
Screenshot 2024-09-26 at 9 46 36 PM
Screenshot 2024-09-26 at 9 48 33 PM
Screenshot 2024-09-26 at 9 56 32 PM

**Comment by hanneskuettner** [comment]
Sorry, I just realized there is a `v-text-overflow` component that can be used instead of the style mixin and fulfills the same functionality without introducing new concepts. Do you mind updating the PR to use that component instead?

**Comment by Julias0** [comment]
> Sorry, I just realized there is a `v-text-overflow` component that can be used instead of the style mixin and fulfills the same functionality without introducing new concepts. Do you mind updating the PR to use that component instead?

Sure! Will check it out

**Review by paescuj — APPROVED** [review]

**Pull request merged**